app/controllers/api/v8/courses/submissions/last_hour_controller.rb

```diff
@@ -0,0 +1,41 @@
+module Api
+  module V8
+    module Courses
+      module Submissions
+        class LastHourController < Api::V8::BaseController
+          include Swagger::Blocks
+
+          swagger_path '/api/v8/courses/{course_id}/submissions' do
+            operation :get do
+              key :description, 'Returns submissions to the course in the latest hour'
+              key :produces, ['application/json']
+              key :tags, ['point']
+              parameter '$ref': '#/parameters/path_course_id'
+              response 403, '$ref': '#/responses/error'
+              response 404, '$ref': '#/responses/error'
+              response 200 do
+                key :description, 'Submission ids'
+                schema do
+                  key :type, :array
+                  items do
+                    key :type, :integer
+                  end
+                end
+              end
+            end
+          end
+
+          def index
+            course = Course.find(params[:course_id])
+            authorize! :read, course
+            return respond_access_denied unless current_user.administrator?
```

Comment:
Should this admin check be at the beginning of the method since nothing is done with the course if the user isn't an admin?